### PR TITLE
Fixes in contacts sync withcozy- vcard update.

### PR DESCRIPTION
**Pull request by jacquarg**
Also fix carddav updates not working.

### CODE REVIEW

File: server/backends/carddav.coffee [hunk 1]
```diff
@@ -85,13 +84,14 @@ module.exports = class CozyCardDAVBackend
             data = @Contact.parse cardData
             data.id = contact._id
             data.carddavuri = cardUri
-
+            # Surprinsingly updateAttributes has no effect without this pre-fill
```
Comment (jsilvestre) [inline server/backends/carddav.coffee]: Can you add in this comment "@TODO: fix during cozydb migration" please?

**Comment by jsilvestre** [comment]
It looks good to me.

**Comment by jsilvestre** [comment]
@jacquarg can you add the build to the PR please?

**Comment by jsilvestre** [comment]
The build seems to be broken here.

**Comment by frankrousseau** [comment]
Thank you @jacquarg and @jsilvestre!

**Pull request merged**